kv/txn_test.go

```diff
@@ -215,28 +181,22 @@ func verifyUncertainty(concurrency int, maxOffset time.Duration, t *testing.T) {
 				t.Fatal(err)
 			}
 
-			if err := txnDB.InternalKV().RunTransaction(txnOpts, func(txn *client.Txn) error {
+			if err := txnDB.Tx(func(tx *client.Tx) error {
 				// Read within the transaction.
-				gr := proto.GetResponse{}
-				if err := txn.Run(client.Call{
-					Args: &proto.GetRequest{
-						RequestHeader: proto.RequestHeader{
-							Key:       key,
-							Timestamp: futureTS,
-						},
-					},
-					Reply: &gr,
-				}); err != nil {
+				tx.SetTimestamp(futureTS.GoTime())
```

Comment:
This part of the test is just verifying that within the same txn, you never run into uncertainty interval errors. I don't see this is a particularly useful or realistic part of the test. @tschottdorf ?